README.md

```diff
@@ -297,12 +289,23 @@ firewall { '002 drop NEW external website packets with FIN/RST/ACK set and SYN u
   state     => 'NEW',
   action    => 'drop',
   proto     => 'tcp',
-  sport     => ['! http', '! 443'],
+  sport     => ['! http', '443'],
```

Comment:
Doesn't seems to be good to have inversion for first value only, user can make mistake by putting inversion on each values. Coz user is dump and expect them to over read the inversion guidelines.
I think the earlier design was good where user need to put inversion for all values. Open for thoughts.

Reply:
I had done this mostly for convenience and to mimic more closely how firewall actually behaves, it seemed wasteful to me to have a bunch of extra negations in the array.
I could implement code so that it accepts either behaviour.
Had also considered simply having them add an exclamation as an array variable, i.e. ` ['!', 'http', '443']`

Reply:
Have updated the code to allow for either behaviour, the relevant array attributes must either have only the first value negated, or be negated in their entirety.